### PR TITLE
Add notes about Privacy settings in Mac OS X 10.9

### PR DESCRIPTION
**Pull request by j2bbayle**
The Security & Privacy settings have been modified in Mac OS X 10.9. I added documentation about it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -293,8 +293,13 @@ To use `subliminal-test`, first:
 2. 	"Share" the "Integration Tests" scheme to make it available to the CI server: 
 	in Xcode, click "Product" -> "Schemes" -> "Manage Schemes…", click the "Shared" 
 	checkbox next to the scheme, and check the resulting file into source control.
-3. 	Enable GUI scripting: Open System Preferences and check "Enable Access for 
-	Assistive Devices in the Accessibility" preference pane.
+3.  Enable GUI scripting: if the test machine is running Mac OS X 10.8 Mountain
+	Lion, open System Preferences and check "Enable Access for Assistive Devices"
+	in the Accessibility preference pane.
+	If the test machine runs Mac OS X 10.9 Maverick, you first have to run the
```
Comment (wearhere) [inline README.md]: Do you actually have to run the script first, in order to add the Terminal to the Security & Privacy panel?

Comment (j2bbayle) [inline README.md]: Unfortunately yes, otherwise the Terminal does not appear in the list.

**Comment by j2bbayle** [comment]
In fact one can drag `Terminal.app` from the Application/Utilities folder to the Accessibility list, you were right @wearhere. I updated the files to reflect that finding.

**Comment by wearhere** [comment]
Thanks @j2bbayle!

**Pull request merged**